File: z-clients/javascript/src/models/svixPollerDeleteOut.ts
// this file is @generated

export interface SvixPollerDeleteOut {
    topic: string;
    pollerId: string;
    success: boolean;
}

export const SvixPollerDeleteOutSerializer = {
    // biome-ignore lint/suspicious/noExplicitAny: intentional any
    _fromJsonObject(object: any): SvixPollerDeleteOut {
        return {
            topic: object['topic'],
            pollerId: object['poller_id'],
            success: object['success'],
        };
    },

    // biome-ignore lint/suspicious/noExplicitAny: intentional any
    _toJsonObject(self: SvixPollerDeleteOut): any {
        return {
            'topic': self.topic,
            'poller_id': self.pollerId,
            'success': self.success,
        };
    }
}

File: z-clients/rust/src/models/svix_poller_delete_out.rs
// this file is @generated
use serde::{Deserialize, Serialize};

#[derive(Clone, Debug, Deserialize, Serialize)]
pub struct SvixPollerDeleteOut {
    pub topic: String,

    pub poller_id: String,

    pub success: bool,
}

impl SvixPollerDeleteOut {
    pub fn new(topic: String, poller_id: String, success: bool) -> Self {
        Self {
            topic,
            poller_id,
            success,
        }
    }
}
